library/Freckle/App.hs

```diff
@@ -225,21 +224,19 @@ newtype AppT app m a = AppT
     , MonadThrow
     , MonadCatch
     , MonadMask
-    , MonadLogger
-    , MonadLoggerIO
     , MonadResource
     , MonadReader app
     )
+  deriving (MonadLogger, MonadLoggerIO) via WithLogger app (ResourceT m)
 
 instance PrimMonad m => PrimMonad (AppT app m) where
   type PrimState (AppT app m) = PrimState m
 
   -- This should really just be `lift . primitive`, but:
   --
   -- - We'd need `MonadTrans (AppT app)`, which meh
-  -- - We'd need an orphan `Primitive LoggingT`, which no thanks
```

Comment:
Yesss!

Maybe we should give `MonadTrans (AppT app)` now :thinking:

Reply:
I don't see why not